src/sentry/integrations/discord/requests/base.py

```diff
@@ -110,3 +110,6 @@ def _error(self, key: str) -> None:
 
     def is_ping(self) -> bool:
         return self._data.get("type", 0) == 1
+
+    def is_command(self) -> bool:
+        return self._data.get("type", 0) == 2
```

Comment:
If we support more than the ping/command data types, we may want to define a separate enum with the data types and replace this with something like 
```suggestion
        return self._data.get("type", 0) == DISCORD_DATA_TYPE.COMMAND
```

Reply:
Don't need to make that change in this PR though

Reply:
Great point. I'll make a note of that!